bindings/generator/templates/client_plugin_definitions.ts.j2

```diff
@@ -32,6 +32,8 @@ string
 Uint8Array
 {%- elif t.kind == "OnClientEventCallback" -%}
 (event: ClientEvent) => void
+{% elif t.kind == "enum"%}
```

Comment:
```suggestion
{%- elif t.kind == "enum" -%}
```

the `-` are needed to strip the newlines (otherwise the `,` end up on the next line)